Sources/ApplicativeRouter/SyntaxRouter.swift

```diff
@@ -171,7 +171,8 @@ private func request(from data: RequestData, base: URL?) -> URLRequest? {
 }
 
 private func urlComponents(from route: RequestData) -> URLComponents {
-  return URLComponents()
-    |> \.path .~ route.path.joined(separator: "/")
```

Comment:
@mbrandonw this lensing crashes Linux :/ May be fixed on the next version, but wonder if we should file a bug?